src/base/static/js/views/app-view.js

```diff
@@ -387,34 +399,38 @@ export default Backbone.View.extend({
       placeParams.page_size = this.options.appConfig.places_page_size;
     }
 
-    // Load places from the API
-    const placeCollectionsPromise = mapseedApiClient.place.get({
-      placeParams,
-      placeCollections: this.places,
-      layers: mapLayersSelector(store.getState()),
-      setLayerError: layerId => store.dispatch(setLayerError(layerId)),
-      hasAdminAbilities: datasetSlug => {
-        return hasAdminAbilities(store.getState(), datasetSlug);
-      },
-    });
+    const datasetConfigs = datasetConfigsSelector(store.getState());
+    const layerStatuses = mapLayerStatusesSelector(store.getState());
+    await Promise.all(
+      datasetConfigs.map(async config => {
+        const placePagePromises = await mapseedApiClient.place.get({
+          url: `${config.url}/places`,
+          datasetSlug: config.slug,
+          clientSlug: config.clientSlug,
+          placeParams: placeParams,
+          includePrivate: hasAdminAbilities(store.getState(), config.slug),
+        });
+
+        // TODO: This Promise.all() should eventually be replaced with a loop 
```

Comment:
@jalMogo -- so I realized that adding paginated data in this PR won't work after all, since the map is not yet smart enough to respond to changes in the places duck length.

So, after everything, I decided to split the difference: in this PR the API client will return an array of Promises, each representing a page of Place data. As a temporary hack, rather than also refactor the map in this PR, we will `await Promise.all()` and load all the data at once.

In #1173, we will refactor the map to accommodate paginated loading and address the TODO on this line.

Reply:
:+1:  Sounds good